user-guide/Advanced_Modules/Automation_module/FAQ/How_do_I_parse_Correlation_Alarm_Info_data.md

```diff
@@ -37,4 +37,7 @@ for (int i=0; i<amountProperties; i++)
     string propertyValue = parts[21 + i*2 + 1];
     engine.GenerateInformation("Property " + propertyName + " == " + propertyValue);
 }
+
+// in 10.5 and up, use the ID below to refer to the alarm tree
```

Comment:
Could more specific versions be specified here? Usually, we specify both the main and feature release versions where a change is introduced.